cmake/dependencies.cmake

```diff
@@ -122,12 +124,37 @@ function(ov_download_tbb)
                 ENVIRONMENT "TBBROOT"
                 SHA256 "321261ff2eda6d4568a473cb883262bce77a93dac599f7bd65d2918bdee4d75b")
     elseif(APPLE AND X86_64)
+        # build oneTBB 2021.2.1 with OS version 11.4
```

Comment:
in future, we need to consider building for 10.12 to be aligned with OpenVINO itself.